src/components/common/ColonyActions/helpers/mapItemToMessageFormat.tsx

```diff
@@ -87,26 +94,28 @@ export const mapColonyActionToExpectedFormat = (
         actionData.amount,
         getTokenDecimalsWithFallback(colony?.nativeToken.decimals),
       ),
-    // rolesChanged: formattedRolesTitle.roleTitle,
+    rolesChanged: formattedRolesTitle.roleTitle,
     newVersion: actionData.newColonyVersion,
   };
 };
 
 export const mapColonyEventToExpectedFormat = (
   eventName: ColonyAndExtensionsEvents,
   actionData: ColonyAction,
+  eventId?: string,
   colony?: Colony,
 ) => {
-  // const role = item.roles[0];
-
   return {
     amount: (
       <Numeral
         value={actionData.amount ?? 0} // @TODO: getAmount(item.actionType, item.amount)
         decimals={actionData.token?.decimals ?? DEFAULT_TOKEN_DECIMALS}
       />
     ),
-    // ...getColonyRoleSetTitleValues(role?.setTo),
+    ...getColonyRoleSetTitleValues(
+      actionData?.individualEvents as string,
```

Comment:
Optional chaining looks like it's not needed here, and could you provide a comment why the casting to string is safe here (otherwise this could be null or undefined as far as the type is concerned)?